src/components/structural/header/ProjectView.js

```diff
@@ -423,6 +435,27 @@ class Project extends React.Component {
                                     }
                                     <span>&nbsp;</span>Preview
                                 </Button>
+                                <input
```

Comment:
Is there any reason why this input tag is outside of the import Button? It seems to be working fine but I think it should be a child of the Button.